bokehjs/package-lock.json

```diff
@@ -4075,6 +4071,7 @@
       "integrity": "sha512-CWBzXQrc/qOkhidw1OzBTQuYRbfyxDXJMVJ1XNwUHGROVmuaeiEm3OslpZ1RV96d7SKKjZKrSJu3+t/xlw3R9A==",
       "dev": true,
       "license": "Apache-2.0",
+      "peer": true,
```

Comment:
Changes to this file must be reverted. I would advice to use `git add -p` to avoid adding unrelated changes to your PRs.